src/main/java/io/split/android/client/service/mysegments/MySegmentsSyncTask.java

```diff
@@ -69,10 +77,18 @@ private void logError(String message) {
         return nameList;
     }
 
+    private @Nullable Map<String, String> getHeaders() {
+        if (mAvoidCache) {
+            return SplitHttpHeadersBuilder.noCacheHeaders();
+        }
+        return null;
+    }
+
     private void fireMySegmentsUpdatedIfNeeded(List<String> oldSegments, List<String> newSegments) {
         if(mEventsManager == null) {
             return;
         }
+        SplitInternalEvent e = getInternalEvent(oldSegments, newSegments);
```

Comment:
`e` is not being used. should it be removed or some logic is missing?

Reply:
Removed!